frontend/src/pages/groups/GroupEditPage.tsx

```diff
@@ -0,0 +1,94 @@
+import { FunctionComponent } from 'react'
+import { gql, useQuery } from '@apollo/client'
+import { useParams } from 'react-router-dom'
+import LayoutWithNav from '../../layouts/LayoutWithNav'
+import { Group, GroupInput } from '../../types/api-types'
+import GroupForm from './GroupForm'
+
+const ALL_GROUP_FIELDS = gql`
+  fragment AllGroupFields on Group {
+    id
+    name
+    groupType
+    primaryLocation {
+      townCity
+      countryCode
+      openLocationCode
+    }
+    primaryContact {
+      name
+      email
+      whatsApp
+      phone
+      signal
+    }
+  }
```

Comment:
Yeah especially as they get longer having them in their own files seems preferable.